telemetry/specs/INT.mdk

```diff
@@ -1342,6 +1339,9 @@ INT-MX Header:
 [^telem-report]: [Telemetry Report Format Specification Version 2.0](https://github.com/p4lang/p4-applications/blob/master/docs/telemetry_report_v2_0.pdf), May 2020.
 
 * **Ver (4b):** INT-MX header version. Should be 2 for this version.
+
+[]{tex-cmd: "\newpage"}
+
```

Comment:
"Bits 0 - 14 are Baseline INT Instructions." at the end of Instruction Bitmap seems not needed. I presume the quoted text was added for INT-MD which has Bit15, checksum complement.

Reply:
It is baseline metadata compared to the domain specific metadata. In the Telemetry Report with RepType INT, the baseline metadata goes into "Variable Optional Baseline Metadata" and the domain specific metadata goes into "Variable Optional Domain Specific Metadata". I prefer keeping it.

Reply:
Done